modules/metagetta/src/cljdoc_analyzer/metagetta/clojurescript.clj

```diff
@@ -117,6 +125,21 @@
 (defn- ns-merger [val-first val-next]
   (update val-first :publics #(seq (into (set %) (:publics val-next)))))
 
+(defn- get-js-package-dependencies
+  "Compute the set of all js dependencies used by a package from its root path.
+
+  Example: for the package 'lilactown-hx-0.5.2', #{\"react\"} is returned."
+  [package-root-path]
+  (->> (find-files package-root-path)
+       (map (fn [file]
+              (->> file
+                   (io/file package-root-path)
+                   (ana/parse-ns)
+                   :requires)))
+       (reduce clojure.set/union)
+       (filter string?)
+       (into #{})))
```

Comment:
Love how readable this is! Maybe we should name this `get-string-requires` just to be a little bit more precise? Docstring could mention that this usually refers to JS dependencies.